apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx

```diff
@@ -937,7 +937,7 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
                 backChannelLogoutUrl: values.get("backChannelLogoutUrl"),
                 frontChannelLogoutUrl: values.get("frontChannelLogoutUrl")
             },
-            publicClient: values.get("supportPublicClients")?.length > 0,
+            publicClient: !isMobileApplication ? values.get("supportPublicClients")?.length > 0 : true,
```

Comment:
```suggestion
            publicClient: isMobileApplication || values.get("supportPublicClients")?.length > 0,
```

Reply:
+1 or use `Nullish coalescing operator (??)` to simplify the ternary.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing

Reply:
Both methods are correct.

Reply:
Both ways are functionally correct. The suggestion was to improve readability :)